front/pages/poke/[wId]/index.tsx

```diff
@@ -314,6 +324,37 @@ const WorkspacePage = ({
     }
   );
 
+  const { submit: onToggleFeature, isSubmitting: isTogglingFeature } =
+    useSubmitFunction(async (feature: WhitelistableFeature, value: boolean) => {
+      try {
+        const r = await fetch(
+          `/api/poke/workspaces/${owner.sId}/features/${feature}`,
+          {
+            method: value ? "POST" : "DELETE",
+            headers: {
+              "Content-Type": "application/json",
+            },
+          }
+        );
+        if (!r.ok) {
+          throw new Error("Failed to disable feature.");
+        }
+
+        await mutate(`/api/poke/workspaces/${owner.sId}/features`);
+      } catch (e) {
+        console.error(e);
```

Comment:
✂️

Reply:
This is Poke, I think having the full error in console has value no ?

Reply:
I thoughts it was an oversight. Even though it's not ideal, I don't mind.

Reply:
I really don't care that much about it, happy to remove 👍